backend/ddang/src/main/java/com/ddang/ddang/auction/domain/Auction.java

```diff
@@ -147,4 +148,26 @@ private BidPrice calculateNextMinimumBidPrice() {
         final int nextMinimumBidPrice = this.lastBid.getPrice().getValue() + this.bidUnit.getValue();
         return new BidPrice(nextMinimumBidPrice);
     }
+
+    public boolean isWinner(final User user, final LocalDateTime targetTime) {
+        final Optional<User> nullableWinner = findWinner(targetTime);
+
+        return nullableWinner.filter(user::equals)
+                             .isPresent();
+    }
+
+    public Optional<User> findWinner(final LocalDateTime targetTime) {
+        if (isWinnerExist(targetTime)) {
+            return Optional.of(lastBid.getBidder());
+        }
+
+        return Optional.empty();
+    }
+
+    private boolean isWinnerExist(final LocalDateTime targetTime) {
+        if (auctioneerCount == 0) {
+            return false;
+        }
+        return isClosed(targetTime);
```

Comment:
```suggestion
        return auctioneerCount != 0 && isClosed(targetTime);
```

합칠 수 있을 것 같습니다 복잡하지는 않은 조건식이라 굳이 분기처리를 할 필요는 없을 것 같아서..